src/ophyd_async/fastcs/panda/_trigger.py

```diff
@@ -1,12 +1,20 @@
 import asyncio
 from typing import Optional
 
+import numpy as np
 from pydantic import BaseModel, Field
+from scanspec.specs import Frames, Path, Spec
 
-from ophyd_async.core import TriggerLogic, wait_for_value
+from ophyd_async.core import (
+    AsyncStatus,
+    TriggerLogic,
+    WatchableAsyncStatus,
+    wait_for_value,
+)
+from ophyd_async.epics import motor
```

Comment:
We shouldn't import `epics` from `fastcs`. Can we use `INENC1.VAL.Scale` from PandA instead?